package.json

```diff
@@ -16,8 +16,7 @@
   "repository": "https://github.com/pycom/pymakr-atom",
   "license": "GPL-3.0",
   "engines": {
-    "atom": ">=1.41.0",
-    "node": ">=6.3.0 <=7.0.0"
+    "atom": ">=1.41.0"
```

Comment:
I think better to keep the least value of the node, what do you think?

Reply:
I'm good either way, but we definitely need to get rid of the `<=7.0.0`, unless I'm misunderstanding its meaning.